grails-app/services/au/org/ala/volunteer/ExportService.groovy

```diff
@@ -552,4 +557,149 @@ class ExportService {
+                        ],
+                        "catalogNumber": "",
+                        "recordedBy": "",
+                        "occurrenceRemarks": ""
+                ],
+                "dwc:Identification": [
+                        "identificationRemarks": ""
+                ],
+                "dwc:Taxon": [
+                        "scientificName": "",
+                        "genus": "",
+                        "specificEpithet": "",
+                        "scientificNameAuthorship": "",
+                        "taxonRank": "",
+                        "infraspecificEpithet": "",
+                        "clazz": ""
+                ],
+                "dwc:Event": [
+                        "eventDate": "",
+                        "verbatimEventDate": "",
+                        "samplingProtocol": "",
+                        "fieldNumber": ""
+                ],
+                "dwc:Location": [
+                        "country": "",
+                        "stateProvince": "",
+                        "locality": "",
+                        "verbatimLocality": "",
+                        "decimalLatitude": "",
+                        "decimalLongitude": "",
+                        "verbatimLatitude": "",
+                        "verbatimLongitude": "",
+                        "minimumElevationInMeters": "",
+                        "maximumElevationInMeters": "",
+                        "verbatimElevation": "",
+                        "town": "",
+                        "habitat": ""
+                ]
+        ]
+
+        List jsonTaskList = []
+
+        taskList.each { Task task ->
+            def jsonMapValues = SerializationUtils.clone(jsonMap) as LinkedHashMap
+
+            def taskFields = (taskMap.containsKey(task.id) ? taskMap[task.id] : null)
+            if (taskFields) {
+                taskFields.each { transcriptionId, taskTranscription ->
+
+                    taskTranscription.each { String fieldName, field ->
+                        // if field ID is in the json map, add it. If multiple values, put them as a list
+
+                        def count = field.size()
+                        for (int i = 0; i < count; i++) {
+                            if (!StringUtils.isEmpty(field[i])) {
+                                searchAndSaveValue(jsonMapValues, fieldName, field[i] as String)
+                            }
+                        }
+                    }
+                }
+            }
+
+            // Check Event Date's validity
+            jsonMapValues["dwc:Event"].verbatimEventDate = jsonMapValues["dwc:Event"].eventDate
+            // Check if the date parses for an ISO 8601 format. If not, wipe the eventDate field (as it's invalid).
+            if (jsonMapValues["dwc:Event"].eventDate ==~ /^\d{4}(?:-\d{1,2}(?:-\d{1,2}))(?:\/\d{4}(?:-\d{1,2}(?:-\d{1,2})))$/) {
+                log.info("Regex parse true: ${jsonMapValues["dwc:Event"].eventDate}")
```

Comment:
Should the `exportJson` and `searchAndSaveValue` `log.info`s be changed to `log.debug` now that development is done?